### ISSUE TITLE
Improve webpage

### ISSUE THREAD

**Issue by cdeil** [gammapy/gamma-cat#204]
@pdeiml - I see that we now have this thanks to your additions:

* https://gamma-cat.readthedocs.io/use/publication_list.html
* https://gamma-cat.readthedocs.io/use/source_list.html

I tried to use it for the first time to look up some infos, and have some suggestions:

- change the links so that clicking doesn't download, but display the data (e.g. on the webpage or just link to the file on Github). Download can maybe be useful as well, but then in addition, usually clicking on things should show the info in the browser
- the publication list isn't very useful. Just linking to the corresponding folder on Github would be very helpful.
- I think adding a link for each source to gamma-sky.net would be good. Example: http://gamma-sky.net/#/cat/tev/25 I don't think we'll have interactive stuff in the gamma-cat webpage, so linking to gamma-sky.net is useful. once we have better content here, we can re-discuss how to avoid duplication, but for now, cross-linking is useful for us and users.

@pdeiml - Thoughts?

**Comment by pdeiml** [comment]
* I can add a link to the github-page, no problem. But download without cloning the whole repo should be possible as well. Can you tell me how to tell sphinx that the data should be shown in the browser and not to be downloaded?

* My plan of the publication list: Click on a reference_id and a new page opens where we provide more information about the reference. First of all the corresponding data. Another nice information would be author, year and title but would need a lot of time because this can not be done via python scripts, I think, it must be inserted by hand. Or we create a database with all information about the references and fill the webpage from that via python but then the information has to be written by hand as well.

* You mean a link to the github-repo or to the webpage itself? I think to the webpage itself, no?

**Comment by cdeil** [comment]
We discussed this on Skype just now, I think it's clear. I'd suggest to do a minimal solution that surfaces the relevant data / files here, and then if you have more time, I would consider #193 and #198 high priority.

To see the download behaviour of the links via a webserver locally:
```
cd docs/_build/html
python -m http.server
# visit localhost in your browser
```

**Comment by cdeil** [comment]
The suggestions above were implemented in #208
